CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add `getNextElement`, `getPreviousElement` and `focusAsync` to exported as `FocusZoneUtilities` @layershifter ([#981](https://github.com/stardust-ui/react/pull/981))
 - Add `Reaction` and `ReactionGroup` components @mnajdova ([#959](https://github.com/stardust-ui/react/pull/959))
 - Add `reactionGroup` and `reactionGroupPosition` props to the `ChatMessage` component @mnajdova ([#959](https://github.com/stardust-ui/react/pull/959))
+- Add screener tests with steps for `Menu` @Bugaa92 ([#983](https://github.com/stardust-ui/react/pull/983))
```

Comment:
I don't see any user facing changes there 🤔

Reply:
not really but we've been adding CHANGELOG entries for this type of work before, I'm ok with removing

Reply:
https://github.com/stardust-ui/react/pull/911#discussion_r257390850
It looks like the same situation...

Reply:
exactly, we need more consistency here, removed